packages/cli/src/utils/runner.ts

```diff
@@ -19,7 +19,7 @@ export const stop = async (lcdEndpoint: string, orchestratorEndpoint: string, mn
 }
 
 const waitForInstance = async (lcd: API, hash: string, maxRetries: number = 10, delay: number = 1, count: number = 0): Promise<IInstance> => {
```

Comment:
```suggestion
const waitForInstance = async (lcd: API, hash: string, maxRetries: number = 60, delay: number = 1000, count: number = 0): Promise<IInstance> => {
```

the `delay` is in millisecond, not second.
I also increase the `maxRetries` to give 60sec for the service to start by default.